brew/src/main/java/nl/tudelft/serg/evosql/brew/generator/junit/JUnitGeneratorHelper.java

```diff
@@ -0,0 +1,176 @@
+                .addParameter(String.class, "query")
+                .addParameter(TypeName.BOOLEAN, "isUpdate")
+                .addException(SQLException.class)
+                .addJavadoc("Executes an SQL query on the database.\n\n" +
+                        "@param query    The query to execute.\n" +
+                        "@param isUpdate Whether the query is a data modification statement.\n\n" +
+                        "@returns The resulting table, or null if the query is an update.\n");
+
+        runSql.addStatement(
+                "$T connection = $T.getConnection($L, $L, $L)",
+                Connection.class, DriverManager.class,
+                NAME_DB_JDBC_URL, NAME_DB_USER, NAME_DB_PASSWORD)
+                .addStatement("$T statement = connection.createStatement()", Statement.class)
+                .beginControlFlow("if (isUpdate == true)")
+                .addStatement("statement.executeUpdate(query)")
+                .addStatement("return null")
+                .nextControlFlow("else")
+                .addStatement("$T tableStructure = new $T()",
+                        RUN_SQL_RETURN_TYPE,
+                        RUN_SQL_RETURN_TYPE)
+                .addStatement("$T resultSet = statement.executeQuery(query)", ResultSet.class)
+                .addStatement("$T columns = $L(resultSet)",
+                        ParameterizedTypeName.get(List.class, String.class), GET_RESULT_COLUMNS_NAME)
+                .beginControlFlow("while (resultSet.next())")
+                .addStatement("$T values = new $T<>()",
+                        ParameterizedTypeName.get(HashMap.class, String.class, String.class), HashMap.class)
+                .beginControlFlow("for (String column : columns)")
+                .addStatement("values.put(column, resultSet.getObject(column).toString())")
+                .endControlFlow()
+                .addStatement("tableStructure.add(values)")
+                .endControlFlow()
+                .addStatement("return tableStructure")
+                .endControlFlow();
+
+        return runSql.build();
+    }
+
+    /**
+     * Builds a runSql implementation specification.
+     *
+     * @return A runSql specification.
+     */
+    public MethodSpec buildRunSqlEmpty() {
+        MethodSpec.Builder runSql = MethodSpec.methodBuilder(RUN_SQL_NAME)
+                .addModifiers(Modifier.PRIVATE, Modifier.STATIC)
+                .returns(RUN_SQL_RETURN_TYPE)
+                .addParameter(String.class, "query")
+                .addParameter(TypeName.BOOLEAN, "isUpdate")
+                .addException(SQLException.class)
+                .addJavadoc("This method should connect to your database and execute the given query.\n" +
+                        "In order for the assertions to work correctly this method must return true in the case\n" +
+                        "that the query yields at least one result and false if there is no result.\n\n" +
```

Comment:
This has changed right? The method should now return either null or a hashmap.

Reply:
Yeah I'll fix that one